chat-plugins/trivia.js

```diff
@@ -1580,8 +1577,8 @@ const commands = {
 
 		let questions = triviaData.questions;
 		let questionID = toId(question);
-		for (let i = 0; i < questions.length; i++) {
-			if (toId(questions[i].question) === questionID) {
+		for (const [i, question] of questions.entries()) {
```

Comment:
`questtion.entries` isn't an array of `[i, question]`

Reply:
Assuming it's an array: yes it is? You know what `array.entries()` does, don't you?

Reply:
...sorry I wan't reading it right